samza-test/src/main/resources/hello-stateful-world.samza

```diff
@@ -18,7 +18,7 @@
 
 # Job
 job.factory.class=samza.job.local.ThreadJobFactory
-job.name=hello-stateful-world
+app.name=hello-stateful-world
```

Comment:
Perhaps you need to update shouldStopNewProcessorsJoiningGroupWhenNumContainersIsGreaterThanNumTasks
which is failing due to this change.

Reply:
Failing test is not related to this change